packages/test/test-version-utils/src/versionUtils.ts

```diff
@@ -589,3 +589,61 @@ export function versionHasMovedSparsedMatrix(version: string): boolean {
 		version >= "2.0.0-internal.2.0.0" || (!version.includes("internal") && version >= "2.0.0")
 	);
 }
+
+
+/**
+ * @internal
+ */
+export function versionToComparisonNumber(version: string): number {
```

Comment:
@tylerbutler , do we already have something like this?

Reply:
What's the intended use of this? Could you add some docs?